UPGRADE.md

```diff
@@ -1,3 +1,14 @@
+### UPGRADE FROM 0.15.0 TO 0.16.0
+
+1. `orderNumber` field on `Sylius\InvoicingPlugin\Entity\Invoice` has been removed and replaced with relation to `Order` entity.
+1. `Sylius\InvoicingPlugin\Entity\InvoiceInterface::orderNumber` function is left due to easier and smoother upgrades,
+   but is also deprecated and will be removed in the `v1.0.0` release. Use `Sylius\InvoicingPlugin\Entity\InvoiceInterface::order` instead.
```

Comment:
Shouldn't we remove that method earlier than v1.0.0?

Reply:
I chose `v1.0.0` because I had no better idea which version it should be :D alpha? beta? rc? I think we can rethink that, but also we can just change this decision later, so it does not block the merge 💃